src/ng/directive/input.js

```diff
@@ -2142,7 +2142,7 @@ var ngValueDirective = function() {
    *  makes it possible to use ngValue as a sort of one-way bind.
    */
   function updateElementValue(element, attr, value) {
-    element.prop('value', value);
+    element.prop('value', angular.isUndefined(value) ? null : value);
```

Comment:
I will update the PR with the `msie` check and using `isDefined` instead of `isUndefined`. 

If I get this straight, using `element[0].value = propValue` should be possible aswell? The difference is we're not using jQuery/jqLite in this case, which may be a better idea but I have no idea what you guys prefer...